src/board.js

```diff
@@ -9,22 +9,232 @@
   }
 }(this, function (exports) {
 
+  // FIXME: implement a quick Set polyfill.
+
+  // A given intersection on a board.
+  function Point(x, y) {
+    this.x = x;
+    this.y = y;
+    this.color = Board.EMPTY;
+    this.group = null;
+    // FIXME: set the following values correctly.
+    this.wasJustCaptured = false;  // Did a capture just happen here?
+    this.turnsSinceLastMove = 0;
+    // Change in number of own/enemy liberties from making a move here.
+    this.ownLibertiesChange = 0;
+    this.enemyLibertiesChange = 0;
+    this.capturesFromMove = 0;  // Number of enemy stones it would capture.
+    this.selfAtariFromMove = 0;  // Number of own stones it would capture.
```

Comment:
The Nature AlphaGo paper is unclear, but my guess is that this is the algorithm they copied: [Pachi self-atari](https://github.com/pasky/pachi/blob/master/tactics/selfatari.h#L59).